spec/features/user_submits_question_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+feature 'user submits a question', %{
+  As an indecisive user
+  I want to submit a question
+  So that I can get help with a decision
+
+  Acceptance Criteria:
+  - [X] I will see a field to enter my question on the homepage
+  - [ ] If I submit an invalid input, I will be notified of errors
+  - [ ] Upon successfully entering my question, I will be notified that my question was submittted
+  - [ ] I will be redirected to a page that shows my question and a form to enter potential answer choices
+} do
+  context "User visits homepage and" do
+    scenario "successfully submits question on homepage" do
+      visit root_path
+      question = "What should I have for lunch?"
+
+      fill_in 'Question', with: question
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.